src/development/LocalTest/Services/Storage/Implementation/ApplicationRepository.cs

```diff
@@ -34,25 +40,15 @@ public Task<bool> Delete(string appId, string org)
             throw new NotImplementedException();
         }
 
-        public Task<Application> FindOne(string appId, string org)
+        public async Task<Application> FindOne(string appId, string org)
         {
-            string filedata = string.Empty;
-            string filename = GetApplicationPath(string.IsNullOrEmpty(appId) ? appId : appId.Split('/')[1]);
-
-            try
-            {
-                if (File.Exists(filename))
-                {
-                    filedata = File.ReadAllText(filename, Encoding.UTF8);
-                }
-
-                return Task.FromResult(JsonConvert.DeserializeObject<Application>(filedata));
-            }
-            catch (Exception ex)
+            var ret = await _localApp.GetApplicationMetadata(appId);
```

Comment:
I would suggest setting a more descriptive variable name here.

Reply:
Changed it to application